test/pool.test.ts

```diff
@@ -31,6 +37,27 @@ async function callOnNextBlock<T>(fn: () => T): Promise<T> {
 
 const CYCLE_BLOCKS = 10;
 
+async function getPoolSigners(): Promise<PoolSigner[]> {
+  const signers = await buidler.ethers.getSigners();
+  const pool = await deployPool(signers[0]);
+  const poolSigners = signers.map(async (signer: Signer) =>
+    newPoolSigner(pool, signer)
+  );
+  return Promise.all(poolSigners);
+}
+
+interface PoolSigner {
```

Comment:
How about calling it `User`? I’m not happy with the “Signer” part because it has no relation to `Signer`.

Reply:
Perfect name! Fixed